internal/idp_test/util_test.go

```diff
@@ -3,8 +3,22 @@ package idp_test
 import (
 	"bytes"
 	"encoding/json"
+	"fmt"
+
+	"github.com/elimity-com/scim"
 )
 
+func getNewServer(idpName string) (scim.Server, error) {
+	switch idpName {
+	case "okta":
+		return newOktaTestServer(), nil
+	case "azuread":
+		return newAzureADTestServer(), nil
+	default:
+		return scim.Server{}, fmt.Errorf("%q does not have a test server", idpName)
```

Comment:
IMO it's okay to ignore / panic in this case. So I would either use `if` / `else` or something like `panic("unreachable")` here.